.gitignore

```diff
@@ -42,7 +42,8 @@ stamp-h*
 
 # /tests/
 /tests/tests-asn1c-smoke/Makefile.am.libasncodec
-/tests/tests-asn1c-smoke/Makefile.am.sample
+/tests/tests-asn1c-smoke/Makefile.am.asn1convert
```

Comment:
Why Makefile.am.asn1convert? It is not used elsewhere.

Reply:
The `check-asn1c-smoke.sh` script calls `asn1c` compiler, which will produce `/tests/test-asn1c-smoke/Makefile.am.asn1convert`. Even though, nothing is using it, I think we want git to ignore the file